src/components/simulator/result/SimResultChart.tsx

```diff
@@ -69,3 +52,26 @@ export const SimResultChart = ({ state, simulationType }: Props) => {
     </div>
   );
 };
+
+const Loading = () => (
+  <div className="flex h-[400px] items-center justify-center rounded-10 bg-black py-10">
```

Comment:
Their is a shortcut to center a unique item in a container : 
```
<div className="grid place-items-center ...">
```
It'll apply items-center & justify-center